examples/mvn/multi-module/pom.xml

```diff
@@ -19,25 +19,33 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
-        <scala.version>2.12.6</scala.version>
+        <scala.version>2.12.8</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
+        <scapegoat.version>1.3.8</scapegoat.version>
 
         <sonar.projectName>[example] Maven Multi Module</sonar.projectName>
         <sonar.sources>src/main/scala</sonar.sources>
         <sonar.tests>src/test/scala</sonar.tests>
         <sonar.junit.reportPaths>target/surefire-reports</sonar.junit.reportPaths>
         <sonar.sourceEncoding>UTF-8</sonar.sourceEncoding>
-        <sonar.scala.version>2.12</sonar.scala.version>
-        <sonar.scala.scoverage.reportPath>target/scoverage.xml</sonar.scala.scoverage.reportPath>
-        <sonar.modukes>module1,module2</sonar.modukes>
+        <sonar.scala.version>${scala.binary.version}</sonar.scala.version>
+        <sonar.scala.scoverage.reportPath>${project.build.directory}/scoverage.xml</sonar.scala.scoverage.reportPath>
+        <sonar.scala.scapegoat.reportPath>${project.build.directory}/scapegoat.xml</sonar.scala.scapegoat.reportPath>
```

Comment:
Does this need extra indentation to align it with the other settings?

Reply:
Oh, they are indented.
I just noticed that in the diff splitted view they look like in the beginning of the line. But that is because that lines are too long that the viewer split it after the spaces, giving the impression that they are after a blank line and with out indentation.
You can view the raw file [here](https://github.com/BalmungSan/sonar-scala/blob/adding-issues-to-maven/examples/mvn/multi-module/pom.xml#L32).